src/app/lang/en.ts

```diff
@@ -1,6 +1,8 @@
+import type Localization from "./localization";
+
 export default {
   common: {
-    name: "Predictive Text Studio",
+    app_name: "Predictive Text Studio",
```

Comment:
There were two keys here labeled `name`, and I only realized this after changing this to TypeScript!